ios/FluentUI/Command Bar/CommandBarButton.swift

```diff
@@ -135,6 +139,28 @@ class CommandBarButton: UIButton {
                         dark: Colors.primary(for: window))
     }
 
+    private var accentImageView: UIImageView?
+
+    private func updateAccentImage(_ accentImage: UIImage?) {
+        if accentImage != accentImageView?.image {
+            if let accentImage = accentImage {
+                accentImageView = UIImageView(image: accentImage)
+                accentImageView?.translatesAutoresizingMaskIntoConstraints = false
+                if let accentImageView = accentImageView,
+                    let imageView = imageView {
+                    insertSubview(accentImageView, belowSubview: imageView)
+                    NSLayoutConstraint.activate([
+                        accentImageView.centerXAnchor.constraint(equalTo: imageView.centerXAnchor),
```

Comment:
nothing guarantee that accentimageview is the same size as imageview. right? is center X anchor good enough?

Reply:
I think constraining the center X and center Y anchors of the two image views is the best way can handle this without a lot of complicated resizing logic. Trying to handle cases where images have different scale ratios could get messy very fast. I think the responsibility should be the application to provide images of the same size.